releasenotes/notes/0.18/replace-builder-program-ecfc438b1cb19339.yaml

```diff
@@ -39,3 +39,5 @@ deprecations:
     now being returned by the pulse builder  doesn't support the ``.insert`` method
     (and there is no insert syntax in the builder). The use of timeslot placeholders
     to block the insertion of other instructions is no longer necessary.
+    The returned :class:`~qiskit.pulse.ScheduleBlock` can be converted to
```

Comment:
Does anything need done with this specifically to get it to the qiskit.org release notes? I remember I needed to do something to the main github.com/qiskit/qiskit repo with changing a release note at some point.

Reply:
We need to update the `release_notes.rst` file here https://github.com/Qiskit/qiskit/blob/master/docs/release_notes.rst for it to be reflected on the hosted version

Reply:
Will do as a follow-up when this is merged.